client/my-sites/comments/comment-list/index.jsx

```diff
@@ -176,14 +151,13 @@ export class CommentList extends Component {
 				>
 					{ map( filteredComments, comment =>
 						<CommentDetail
-							commentId={ comment.ID }
-							deleteForever={ this.deleteForever( comment.ID ) }
+							comment={ comment }
+							deleteCommentPermanently={ this.deleteCommentPermanently( comment.ID ) }
```

Comment:
Not a blocker, but we can avoid the function creation here by updating by letting the `<CommentDetail />` component call this with:

```
this.props. deleteCommentPermanently( this.props.commentId )
```